share/doc/homebrew/Prose-Style-Guidelines.md

```diff
@@ -0,0 +1,77 @@
+
+Homebrew's audience includes users with a wide range of education and experience, and users for whom English is not a native language. We aim to support as many of those users as feasible.
+
+We strive for "correct" but not "fancy" usage.
+
+This is a set of guidelines to be applied using human judgment, not a set of hard and fast rules. It is like [The Economist's Style Guide](http://www.economist.com/styleguide/introduction) or [Garner's Modern American Usage](https://en.wikipedia.org/wiki/Garner's_Modern_American_Usage). It is less like the [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide). All guidelines here are open to interpretation and discussion. 100% conformance to these guidelines is *not* a goal.
+
+The intent of this document is to help authors make decisions about clarity, style, and consistency. It is not to help settle arguments about who knows English better. Don't use this document to be a jerk.
+
+## Guidelines
+
+We prefer:
+
+### Style and usage
+
+* British/Commonwealth English over American English, in general
+* "e.g." and "i.e.": Go ahead and use "e.g." or "i.e." instead of spelling them out. Don't worry about putting a comma after them.
+* Offset nontrivial subordinate clauses with commas
+
+### Personal pronouns
+
+* We respect all people's choice of personal pronouns
+* Singular "they" when someone's gender is unknown
+* Avoid gender-specific language when not necessary
+
+### Structure and markup
+
+* Sentence case in section headings, not Title Case
+* Periods at the ends of list items where most items in that list are complete sentences
+* More generally, parallel list item structure
+* Capitalize all list items if you want, even if they're not complete sentences; just be consistent within each list, and preferably, throughout the whole page
+* Use a subordinate list item instead of dropping a multi-sentence paragraph-long item into a list of sentence fragments
+* GitHub flavored Markdown. GitHub's implementation is the standard, period.
+
+### Typographical conventions
+
+* Literal text in commands and code is styled in `fixed width font`
+* Placeholders inside code snippets are marked up with `<...>` brackets
+ * e.g. `git remote add <my-user-name> https://github.com/<my-user-name>/homebrew-core.git`
+* No "$" with environment variables mentioned outside code snippets
+ * e.g. "Set `BLAH` to 5", not "Set `$BLAH` to 5"
+* One space after periods, not two
+* Capitalized proper nouns
+* We do not defer to extensive nonstandard capitalization, typesetting, or other styling of brand names, aside from the normal capitalization of proper nouns and simple internal capitalization
+* No "TM", &trade;, <sup>SM</sup>, &copy;, &reg;, or other explicit indicators of rights ownership or trademarks; we take these as understood when the brand name is mentioned
+
+### Terminology, words, and word styling
+
+* "pull request", not "Pull Request"
```

Comment:
IMO, possibly `Pull request` rather than `pull request` just because it matches GitHub's usage and we actively refer to filed Issues as `Issue #xyz` rather than `issue #xyz`.